libs/ast_generic/AST_generic_helpers.ml

```diff
@@ -119,6 +119,25 @@ let name_of_ids xs =
           name_info = empty_id_info ();
         }
 
+let name_of_name_t name =
```

Comment:
Can you clarify when one would use this vs using `name_t_to_ast_generic_name` below?

Reply:
Changed the names to be more specific